app/assets/javascripts/analytics/_init.js

```diff
@@ -7,5 +7,6 @@
     this.events.init();
     this.virtualPageViews.init();
     this.scrollTracking.init();
+    this.trackExternalLinks.init();
```

Comment:
If we add the new module to the `_init.js` we also need to add it to the spec `manifest.js` (though there may well be a better way of doing this?) otherwise the JS tests will fail.

Reply:
hey @katstevens  - I have updated the tests